locale/locale.bzl

```diff
@@ -0,0 +1,33 @@
+"""A rule to unpack c locale from the debian package."""
+
+def _impl(ctx):
+    ctx.actions.run(
+        executable = ctx.executable._extract,
+        arguments = [
+            ctx.file.deb.path,
+            ctx.outputs.tar.path,
+        ],
+        inputs = [ctx.file.deb],
+        outputs = [ctx.outputs.tar],
+    )
+
+locale = rule(
+    attrs = {
+        "deb": attr.label(
+            allow_single_file = [".deb"],
+            mandatory = True,
+        ),
+        # Implicit dependencies.
+        "_extract": attr.label(
+            default = Label("//locale:extract_locale"),
```

Comment:
I'm really new to Bazel as well, but I guess perhaps it's not really difficult to make this portion general (e.g., by accepting another input attribute to the rule besides `deb` on line 16) and share code between this and `cacerts/cacerts.bzl`. However, let's not attempt to do this now. I'll just file a cleanup issue.

Reply:
Filed #552. This will be useful if we can generalize this.

Reply:
Had the same reflection after writing the code. There should probably be a kind of pkg_tar_from_deb rule with an includes / excludes attribute.